janus_core/md.py

```diff
@@ -657,6 +683,17 @@ def __init__(
             externalstress=self.pressure * units.bar,
         )
 
+    def _parameter_prefix(self) -> str:
```

Comment:
This is using a method to return parameters, the base version in MolecularDynamics just returns "" which is a bit clunky.

The advantage is if in future we extend and want even more parameters for certain ensembles it saves having to add all those variables to base as None's and checking them each